pkgs/by-name/se/serious-sans/package.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "serious-sans";
+  version = "a23f2b3";
```

Comment:
Version should be of format `unstable-YYYY-MM-DD`, using the date of the commit. See pkgs/README.md for a detailed explanation.

Reply:
Fixed, thank you!